docs/specs/openapi.yaml

```diff
@@ -0,0 +1,335 @@
+            Defaults to zero if not specified, in which case the request will wait indefinitely until the change is ready.
+            Read more on the format of the timeout [here](#timeout).
+      responses:
+        '200':
+          description: Change information.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  type:
+                    type: string
+                    enum: ["sync"]
+                  status-code:
+                    type: integer
+                    example: 200
+                  status:
+                    type: string
+                    example: "OK"
+                  result:
+                    $ref: '#/components/schemas/ChangeInfo'
+              examples:
+                start service:
+                  value:
+                    type: sync
+                    status-code: 200
+                    status: OK
+                    result:
+                      id: "6"
+                      kind: "start"
+                      summary: "Start service \"svc1\""
+                      status: "Done"
+                      tasks:
+                        - id: "4"
+                          kind: "start"
+                          summary: "Start service \"svc1\""
+                          status: "Done"
+                          progress:
+                            label: ""
+                            done: 1
+                            total: 1
+                          spawn-time: "2024-12-12T12:50:29.743512342+08:00"
+                          ready-time: "2024-12-12T12:50:30.751092759+08:00"
+                      ready: true
+                      spawn-time: "2024-12-12T12:50:29.743570175+08:00"
+                      ready-time: "2024-12-12T12:50:30.751094634+08:00"
+        '400':
+          description: Bad Request - invalid timeout parameter.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+        '404':
+          description: Not Found - change ID not found.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+        '504':
+          description: Gateway Timeout - timeout waiting for change.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+  /v1/services:
+    get:
+      summary: Get services
+      description: Fetch information about specific services (or all of them), ordered by service name.
+      tags:
+        - services
+      parameters:
+        - in: query
+          name: names
+          description: The names of the services to get, a comma-separated string. If empty, get all services.
+          schema:
+            type: string
+          examples:
```

Comment:
I don't see where these examples show up. Let's just drop them.

Reply:
This is because of one detailed CSS customisation that we shouldn't do, I will revert it and keep the examples, otherwise, a random example will be generated by swagger automatically according to the schema, which makes less sense. However, I will reduce the example to just one.